src/components/Common/SnapshotRedirectButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+
+import { ProposalAttributes } from '../../entities/Proposal/types'
+import { snapshotProposalUrl } from '../../entities/Proposal/utils'
+import OpenExternalLink from '../Icon/OpenExternalLink'
+
+import './SnapshotRedirectButton.css'
+
+interface Props {
+  proposal: Pick<ProposalAttributes, 'snapshot_id' | 'snapshot_space'>
+  inverted?: boolean
+}
+
+const SnapshotRedirectButton = ({ proposal, inverted }: Props) => {
+  return (
+    <Button
+      fluid
+      inverted={inverted}
+      primary={!inverted}
+      href={snapshotProposalUrl(proposal)}
+      target="_blank"
+      rel="noopener noreferrer"
+      className="SnapshotRedirectButton"
+    >
+      <div className="SnapshotRedirectButton__Hidden" />
```

Comment:
What's the purpose of this div?

Reply:
I wasn't able to put the label at the center with the external link icon to the right using absolute positioning, because when hovering over the button, the transformY() changed everything and the icon changed it's position.

The easy way out was using 3 elements, hide one, and let flex do the magic 🪄

Reply:
Maybe `SnapshotRedirectButton__Spacing` or something space-related?